suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -534,6 +534,11 @@ export const selectIsDeviceProtectedByPin = (state: DeviceRootState) => {
     return !!features?.pin_protection;
 };
 
+export const selectIsDeviceProtectedByWipeCode = (state: DeviceRootState) => {
+    const features = selectDeviceFeatures(state);
+    return !!features?.wipe_code_protection;
+};
+
```

Comment:
I would not create a selector just for this, it is only used once and it is very simple. But that's up to (your) personal preference

Reply:
I left this one there for now.